app/views/users/_fields.slim

```diff
@@ -1,5 +1,19 @@
-= f.text_field :name
-= f.text_field :location
-= f.text_field :gender, collection: [:female, :male]
-= f.text_field :interest_list, value: @user.interest_list.join(',')
-= f.submit
+.Form
+  .Form-item
+    = f.label :name
+  .Form-item--input
```

Comment:
1. Let's try to use modifiers only in the main Component
2. This modifier isn't defined in the stylesheet